### ISSUE TITLE
Does not work when nested in a `NavigationView`

### ISSUE THREAD

**Issue by shawna-mac** [swiftui-library/hstack-snap-to-scroll#3]
Hello!  I'm noticing an issue where the library doesn't work when nested in a `NavigationView`.  The views are displayed correctly, however I am unable to scroll only when nested in a `NavigationView`.  I have included a very simple implementation reusing your demo to show what is happening.
[card test.zip](https://github.com/swiftui-library/hstack-snap-to-scroll/files/7534549/card.test.zip)

**Comment by muzammil-triffort** [comment]
@shawna-mac Have you found any solution for this?

**Comment by shawnadrund** [comment]
@muzammil-triffort No.  I just removed the library and wrapped a CollectionView in a representable and made my own that way.
